manifests/ubuntu/agent6.pp

```diff
@@ -16,11 +16,14 @@
   Optional[String] $apt_keyserver = undef,
 ) inherits datadog_agent::params {
 
-  ensure_packages(['apt-transport-https'])
+  exec { 'apt-transport-https':
+    command => '/usr/bin/apt-get install -y -q apt-transport-https'
+  }
+
   if !$skip_apt_key_trusting {
-    ::datadog_agent::ubuntu::install_key { [$apt_key]:
-      server => $apt_keyserver,
-      before => Apt::Source['datadog6'],
+    $key = {
+      'id' => $apt_key,
+      'server' => $apt_keyserver
     }
   }
```

Comment:
I can't remember if it's necessary to do this in case `$skip_apt_key_trusting` is `true`.

```
else  {
  $key = {}
}
```

Reply:
I guess it doesn't hurt to add it, and it is a bit more explicit